homeassistant/helpers/entity.py

```diff
@@ -58,7 +58,11 @@
 FLOAT_PRECISION = abs(int(math.floor(math.log10(abs(sys.float_info.epsilon))))) - 1
 
 
-ENTITY_CATEGORIES_SCHEMA: Final = vol.In(ENTITY_CATEGORIES)
+# pylint: disable-next=invalid-name
+def ENTITY_CATEGORIES_SCHEMA(value: Any | None) -> EntityCategory:
+    """Validate entity category configuration."""
+    value = vol.In(ENTITY_CATEGORIES)(value)
+    return EntityCategory(value)
```

Comment:
I think that this works too?
```suggestion
ENTITY_CATEGORIES_SCHEMA = EntityCategory

Reply:
hmm it doesn't work but I thought it did.

Reply:
No, that doesn't work if we want to allow strings as input to the validator